web/app/styles/components/document/related-resources.scss

```diff
@@ -89,30 +89,24 @@
   }
 
   .button-affordance {
-    @apply flex top-[4px] right-0 rounded p-1;
+    @apply top-[4px] right-0 flex rounded p-1;
     @apply border border-transparent text-color-foreground-primary;
   }
 }
 
 .related-resource-link {
-  @apply relative grid gap-2 items-start rounded border border-transparent py-[5px];
-  @apply w-[calc(100%+12px)] -ml-[8px] pl-[8px] pr-3 -mr-[4px];
+  @apply relative grid items-start gap-2 rounded border border-transparent py-[5px];
+  @apply -ml-[8px] -mr-[4px] w-[calc(100%+12px)] pl-[8px] pr-3;
   @apply outline-offset-[3px];
   grid-template-columns: 22px auto;
 
   .hermes-document + .external-resource {
     @apply mt-1.5;
   }
-
-  .external-resource {
-    .related-resource-link {
-      @apply items-center;
-    }
-  }
```

Comment:
This is the actual change. The rest is Prettier (apologies).

Reply:
💅